packages/quantic/force-app/main/default/lwc/quanticRecentQueries/quanticRecentQueries.html

```diff
@@ -0,0 +1,38 @@
+<template>
+  <lightning-card title={label} class="container">
+    <lightning-button-icon
+      class="slds-p-bottom_small"
+      slot="actions"
+      onclick={toggleVisibility}
+      icon-name={collapseIcon}
+      alternative-text="toggle recent queries visibility"
+      variant="bare"
+    >
+    </lightning-button-icon>
+    
+    <template if:false={isCollapsed}>
+      <template if:false={hasQueries}>
+        <div class="slds-text-body_small empty-list-message slds-p-horizontal_medium">{labels.emptyListLabel}</div>
+      </template>
+      <ul class="slds-m-around_small slds-p-bottom_xxx-small">
+          <template for:each={queries} for:item="query" for:index="index">
+            <li
+              key={query}
+              value={index}
+              onclick={executeQuery}
+              class="slds-grid slds-gutters_direct-xxx-small slds-text-body_small nowrap slds-var-m-bottom_small slds-var-m-left_xx-small"
+            >
+              <span class="slds-col icon-color">
+                <lightning-icon
+                  icon-name="utility:search"
+                  size="xx-small"
+                  class="slds-current-color"
+                ></lightning-icon>
+              </span>
+              <span class="slds-col slds-m-left_xxx-small query-text">{query}</span>
```

Comment:
To make sure longm single-word queries are truncated properly I think you'll need to do something like this.
```suggestion
<span class="slds-col slds-m-left_xxx-small query-text">
  <p class="slds-truncate">{query}</p>
</span>
```
The containing <li> element might need to be given `min-width: 0;` for this to work properly as well.